homeassistant/components/todo/__init__.py

```diff
@@ -320,6 +330,10 @@ def _async_write_ha_state(self) -> None:
         super()._async_write_ha_state()
         self.async_update_listeners()
 
+    async def async_remove_list(self) -> None:
+        """Remove the To-do list."""
+        raise NotImplementedError
+
```

Comment:
Should this function remain unimplemented?

Reply:
> Should this function remain unimplemented?

It is a “pure virtual function” which should be only implemented by subclasses of `TodoListEntity`.
In our case, the implemention is in `homeassistant/components/google_tasks/todo.py`